recipes/python-certifi-win32/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "python-certifi-win32" %}
+{% set version = "1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e074663a1b6e54233f28d15832f9872440472144abafa432ec7f5d473fa33085
+
+build:
+  number: 0
+  skip: true  # [not win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - certifi
```

Comment:
You probably only need pip, setuptools_scm and Python in `host`

Reply:
Fixed https://github.com/conda-forge/staged-recipes/pull/10576/commits/a3b5f4feefa708c4720bd50cf2cb3c4b94e5a5cb